src/components/Footer/Footer.js

```diff
@@ -0,0 +1,83 @@
+import React from 'react';
+import {Col, Container, Row} from 'react-bootstrap';
+import phoneSVG from '../../assets/phone.svg';
+import mailSVG from '../../assets/mail.svg';
+import instagramSVG from '../../assets/instagram.svg';
+
+function Footer() {
+  return (
+    <Container fluid className={'footer mt-4'}>
+      <Row className={'text-center text-md-left pt-4 '}>
+        <Col xs={12} md={3} xl={2} className={' offset-xl-2'}>
+          <h3 className={'font-weight-bold'}>Opening Hours</h3>
+          <ul>
+            <li className={'py-1'}> monday: 9:00-22:00</li>
+            <li className={'py-1'}> tuesday: 9:00-22:00</li>
+            <li className={'py-1'}> wednesday: 9:00-22:00</li>
+            <li className={'py-1'}> thursday: 9:00-22:00</li>
+            <li className={'py-1'}> friday: 9:00-22:00</li>
+          </ul>
+        </Col>
+        <Col xs={12} md={3} xl={2} className={'  '}>
+          <h3 className={'font-weight-bold'}>Locations</h3>
+          <ul>
+            <li>
+              <p>
+                Location 1:
+                <br /> 1475 Spinnaker Lane
```

Comment:
Is it okay to put p tags in li tags?

Reply:
I haven't heard about it being a bad practice but maybe there's someone smart in the web who touched the subject. I don't see a reason why would you not do that if your <li> consists of multiple paragraphs. In other cases why even use <p>?